app/src/main/java/org/eyeseetea/malariacare/layout/listeners/SwipeTouchListener.java

```diff
@@ -43,7 +43,7 @@ public boolean onTouch(View v, MotionEvent event) {
             fixScrollEventY(event);
             scrollView.onTouchEvent(event);
         }
-        return gestureDetector.onTouchEvent(event);
+        return false;
```

Comment:
@manuelplazaspalacio I have a doubt here. This Listener is in the main and affects also the rest of the variants. returning false wouldn't it make the rest of the variants to stop working? The feature I'm concerned about is the previous/next swipe in Laos, for example. I guess that we would need to create a Strategy here, moving the common behavior to a common method and separating the difference into a per-variant answer?

Reply:
@ifoche Finally I add the strategy to not change the behaviour in other variants. I am checking and returning false on onDown method in the swipeTouchListener solve the problem, I don't understand why we  have to consume the motion event. I think we have to review the SwipeTouchListener code for other variants with Android version 4.4.2, because I'm not sure that scroll is working correctly.